### PR TITLE
Add ID_Compat_Math_Start and ID_Compat_Math_Continue.

### PR DESCRIPTION
**Pull request by eggrobin**
Per action item [174-A20](https://www.unicode.org/cgi-bin/GetL2Ref.pl?174-A20).
See [L2/23-008](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/23-008) Section 6.1.

### CODE REVIEW

File: unicodetools/data/ucd/dev/PropertyAliases.txt
Comment (markusicu) [inline unicodetools/data/ucd/dev/PropertyAliases.txt:178]: Question for @Ken-Whistler @macchiati @asmusf: How should this file be sorted? By short or long name? ASCII or UCA or something else? Underscore -- ignore, by code point, like space, ...?

I would move these two to before ID_Continue, but I don't know if it matters.

File: unicodetools/src/main/java/org/unicode/props/UcdProperty.java
Comment (markusicu) [inline unicodetools/src/main/java/org/unicode/props/UcdProperty.java:284]: @eggrobin did you get this from running GenerateEnums?

File: unicodetools/data/ucd/dev/PropList.txt
Comment (markusicu) [inline unicodetools/data/ucd/dev/PropList.txt:1263]: @eggrobin Did you run the UCD generator over the files, or is this a purely manual edit?

File: unicodetools/src/main/java/org/unicode/props/UcdProperty.java
Comment (eggrobin) [reply unicodetools/src/main/java/org/unicode/props/UcdProperty.java:284]: All changes are manual, but I think this one is checked by tests.

File: unicodetools/data/ucd/dev/PropList.txt
Comment (eggrobin) [reply unicodetools/data/ucd/dev/PropList.txt:1263]: All manual.

File: unicodetools/data/ucd/dev/PropertyAliases.txt
Comment (Ken-Whistler) [reply unicodetools/data/ucd/dev/PropertyAliases.txt:178]: I agree they should be ahead of ID_Continue. In general, for this kind of file, I'd treat the underscores like spaces for ordering.

**Comment by macchiati** [comment]
The property aliases and value aliases will be sorted when they are regenerated. So you can enter them in any order but the next time they are generated they'll be reordered.

If we want to change the order in the future, that is possible but probably not worth it (there are lots of cases where the ordering is suboptimal). Part of that is English: if we had noun-adjective(-adjective)* for names then the sort order would be more likely to group connected items together. Sadly, we're not as clever as the French.

**Review by markusicu — APPROVED** [review]
I did another little shuffle and then ran the tools to regenerate the files.
Once the CI tests, I will squash-and-merge.

**Comment by markusicu** [comment]
Merge conflict. Will rebase & resolve.

**Review by markusicu — DISMISSED** [review]
re-approve after rebase

**Review by markusicu — APPROVED** [review]
re-approve 2: I squashed manually, hoping to get the license/cla checker unstuck.

**Comment by eggrobin** [comment]
> squashed manually, hoping to get the license/cla checker unstuck

I think closing/reopening the PR also works to get CI to retry.

**Pull request merged**
